### DISCUSSION TITLE
Errore grave nel registro

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#631] [Richieste di aiuto]
Buonasera io dopo aver effettuato l'aggiornamento non riesco a firmare nn compare la scritta aggiungi e non posso mettere voti

**Comment by Davide031** [comment]
e quando metto le assenze esce error 500
[2024-01-18_15-16.log](https://github.com/iisgiua/giuaschool/files/13977679/2024-01-18_15-16.log)

**Comment by trinko** [comment]
Vedi
[segnalazione di errore](https://github.com/iisgiua/giuaschool/issues/632)

**Comment by Davide031** [comment]
Ho fatto tutto ciò che mi ha detto, tutti i problemi persistono, io ho
fatto il primo bimestre ora non ho ancora chiuso lo scrutinio del primo
trimestre che terminerà a febbraio come devo fare?

Il ven 19 gen 2024, 18:40 Antonello Dessì ***@***.***> ha
scritto:

**Comment by Davide031** [comment]
Se io metto le proposte di voto funziona ma il resto no

Il ven 19 gen 2024, 23:07 DAVIDE COCO ***@***.***> ha scritto:
